### ISSUE TITLE
Plugin to loop through icons and build the sprite.svg

### ISSUE THREAD

**Issue by JestVA** [stephane777/date_picker#3]
Hi Steph, I was looking through your source code and I was a bit confused why you'd have both the svg assets (the 4 icons) in separate files but also the same paths in the sprite.svg file. At first, I thought this is created as a webpack build time or something programatically -- as in looking at your icon lib and populating the sprite file dynamically.  Do you think it is better to delete the redundant files that you have in the SVG folder or either use a plugin like [grunt-svg-store](https://github.com/FWeinb/grunt-svgstore) to loop those ids and create the paths.. `icon-${insert-svg-name}`

**Comment by stephane777** [comment]
Hi Dorin, I have chosen to use svg as it seems to be better [compared](https://css-tricks.com/icon-fonts-vs-svg/) to icon-font. I have used [icomoon](https://icomoon.io/#docs) icon pack which allow me to pick up the svg I want and download all of them and generated in a sprite. The example I have followed let the svg folder in combination with the sprite file in the project so didn't realise it was redundant.
After removing the the svg folder the icons are still loaded ! I will do a commit with the change.
thanks.
